appengine-java21/taskqueues-deferred/README.md

```diff
@@ -1,7 +1,7 @@
 App Engine Java Guestbook
 Copyright (C) 2010-2012 Google Inc.
```

Comment:
can you modify this to be the correct format for showing copyright with just the first year and not the span.

Reply:
Fixed